easybuild/easyconfigs/b/BCL/BCL-4.3.0-GCCcore-11.2.0-Python-2.7.18.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'BCL'
+version = '4.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.meilerlab.org/index.php/bclcommons/show/b_apps_id/1'
+description = """The Bio Chemical Library (BCL) is a software package that provides unique tools for biological
+research, such as protein structure determination from sparse experimental data. The BCL contains the widely used
+secondary structure prediction program JUFO, a folding algorithm that assembles secondary structure elements, and
+loop construction tools that complete protein backbones."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+github_account = "BCLCommons"
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['cebd3214b3f5922af91d0fbc7cf4e85a5878e02ce2f605c8f51f72599638117c']
```

Comment:
I cannot find this to download. https://github.com/BCLCommons/bcl/releases only has a release tagged `public_opening` and the name of one of the files `bcl-4.3.0-Source.tar.gz` suggests this is 4.3.0. However, none of the files there have the same checksum.

Reply:
Looks like they removed the v4.3.0 tag. I will update the easyconfig to use `public_opening` instead. It is not pretty tough